docs/swagger.yaml

```diff
@@ -154,759 +124,845 @@ paths:
       operationId: updatestatusUsingPOST
-      consumes:
-        - application/json
-      produces:
-        - '*/*'
       parameters:
-        - name: Authorization
-          in: header
-          description: Bearer JWT
-          required: true
-          type: string
-        - name: merchant-serial-number
-          in: header
-          description: The Vipps number (MSN) for the merchant's sales unit.
-          required: true
-          type: string
-        - name: Ocp-Apim-Subscription-Key
-          in: header
-          description: 'Subscription key for the PSP API. This is the PSP''s key, and the same for all the PSP''s merchants.'
-          required: true
-          type: string
-        - name: psp-id
-          in: header
-          description: 'PSP id as provided by Vipps. This is the PSP''s id, and the same for all the PSP''s merchants. GUID value for PSPs created after start of 2021.'
-          required: true
-          type: string
-        - in: body
-          name: pspPaymentStatus
-          description: pspPaymentStatus
-          required: true
-          schema:
-            $ref: '#/definitions/PaymentStatus'
+        - $ref: "#/components/parameters/Authorization"
+        - $ref: "#/components/parameters/Merchant-Serial-Number"
+        - $ref: "#/components/parameters/Ocp-Apim-Subscription-Key"
+        - $ref: "#/components/parameters/Psp-Id"
+        - $ref: "#/components/parameters/Vipps-System-Name"
+        - $ref: "#/components/parameters/Vipps-System-Version"
+        - $ref: "#/components/parameters/Vipps-System-Plugin-Name"
+        - $ref: "#/components/parameters/Vipps-System-Plugin-Version"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PaymentStatus"
+        description: pspPaymentStatus
+        required: true
       responses:
-        '200':
+        "200":
           description: OK
-          schema:
-            $ref: '#/definitions/Response'
-        '400':
-          description: Bad Request
-          schema:
-            type: array
-            items:
-              $ref: '#/definitions/ErrorDetails'
-        '401':
+          content:
+            "*/*":
+              schema:
+                $ref: "#/components/schemas/Response"
+        "400":
+          description: Invalid request, check your parameters
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ErrorDetails"
+        "401":
           description: Unauthorized
-        '500':
+        "500":
           description: Internal Server Error
-          schema:
-            type: array
-            items:
-              $ref: '#/definitions/ErrorDetails'
-        '503':
+          content:
+            "*/*":
```

Comment:
all */* should be fixed

Reply:
Do you mean that everywhere it says "*/*", it should be      application/json       ?

Also, I have see it as          application/json;charset=UTF-8    

Should they all include charset=UTF-8?

Reply:
Alf clarified it for me.  Update all of them to application/json. Leave off charset=UTF-8.